core/services/chainlink/application.go

```diff
@@ -291,6 +292,14 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		return nil, fmt.Errorf("could not instantiate workflow rate limiter: %w", err)
 	}
 
+	workflowSyncerLimiter, err := syncerlimiter.NewWorkflowSyncerLimiter(syncerlimiter.Config{
```

Comment:
Just workflowLimiter, rather than workflowSyncerLimiter, it should apply to all workflows, not just those belonging to the workflowSyncer